src/components/Validator/ChecksSection.tsx

```diff
@@ -44,7 +44,7 @@ export function ChecksSection({ checks, isRunning }: ChecksSectionProps) {
                 >
                   <Table.Header>
                     <Table.Row>
-                      <Table.ColumnHeaderCell></Table.ColumnHeaderCell>
+                      <Table.ColumnHeaderCell width="20px"></Table.ColumnHeaderCell>
```

Comment:
Without this, the column width would be distributed equally between all columns making the status icon column needlessly large.